CMakeLists.txt

```diff
@@ -108,8 +108,8 @@ endif()
 #------------------------------------------------------------------
 #Install
 #------------------------------------------------------------------
-
 install(PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/deps/gptp/linux/build/obj/daemon_cl DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(CODE "execute_process(COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/setcap.sh ${CMAKE_INSTALL_PREFIX}/bin/daemon_cl)")
```

Comment:
Maybe use `${CMAKE_INSTALL_BINDIR}/daemon_cl`?

Reply:
it doesn't take the variable in the context.

running setcap on PATH env var
Failed to set capabilities on file `bin/daemon_cl' (No such file or directory)

Reply:
You should probably use the `_FULL` version of the variable then:  CMAKE_INSTALL_FULL_BINDIR`.